alterations/alterations_test.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+    "testing"
+    // "fmt"
+    "github.com/OWASP/Amass/v3/alterations"
+)
+
+var subdomain = "fa2ke.test.com"

```

Comment:
In general, we tend to prefer to use `owasp.org` as a target for tests, but it doesn't really matter either way.

Also, try to keep this information scoped to the tests themselves to avoid taking good names from the package itself (since they are global to the entire package). See my general note to see how this can be shared between tests.

Reply:
Thank you for your advice, I just recently got into Golang so I will implement your suggestions and make another PR

Reply:
You don't have to make a new PR. Just make the changes to this branch and push them up and they'll automatically get added to this PR.